src/test/java/com/iab/gdpr/VendorConsentTest.java

```diff
@@ -133,4 +177,30 @@ public void testRangeEntryConsent() {
         assertFalse(consent.isVendorAllowed(3));
         assertTrue(consent.isVendorAllowed(27));
     }
+    
+    @Test
+    public void testLongRangeEntry() {
```

Comment:
This PR seems to have duplicated this test method, 218a54cfabdf704182cea31d4483b1eba43e05f4 does not build anymore for me.

 @lanusau Did you run the `gradlew test` before merging? Can you add travis integration for this repository? There's already a `.travis.yml` executing the tests by default, so that should be straightforward?

Reply:
My bad. I will fix this. Yes, we need to add travis integration

Reply:
This was fixed in #23, I merged that and it should be passing now.

Reply:
Alright, thanks!